src/Eccube/Resource/template/default/Shopping/shipping_edit.twig

```diff
@@ -22,6 +22,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% extends 'default_frame.twig' %}
 
 {% set body_class = 'cart_page' %}
+{#ログイン状態によって、表示内容を切替#}
+{% set subtitle = '商品購入 / お届け先の変更' %}
+{% set title = 'お届け先の変更' %}
+{% if is_granted('ROLE_USER') %}
+    {% set subtitle = '商品購入 / お届け先の追加' %}
```

Comment:
the other page title is non-whitespace.
(see admin/content/page)
![image](https://cloud.githubusercontent.com/assets/16891862/22869753/a4790b72-f1e4-11e6-861b-f40d85fbbdee.png)

I want to keep it unified.